tb_pulumi/ec2.py

```diff
@@ -101,7 +104,7 @@ def __init__(
         )
 
         # Build the load balancer first, as other resources must be attached to it later
-        nlb = aws.alb.LoadBalancer(
+        nlb = aws.lb.LoadBalancer(
```

Comment:
These are actually the same thing under the hood, such that `aws.lb` is the authoritative library and `aws.alb` is just a reference to it. This leads to no actual change in the resource.